layers/+lang/emacs-lisp/packages.el

```diff
@@ -98,9 +103,47 @@
                srefactor-lisp-format-sexp
                srefactor-lisp-one-line)
     :init
-    (evil-leader/set-key-for-mode 'emacs-lisp-mode
-      "m=b" 'srefactor-lisp-format-buffer
-      "m=d" 'srefactor-lisp-format-defun
-      "m=o" 'srefactor-lisp-one-line
-      "m=s" 'srefactor-lisp-format-sexp
-      )))
+    (dolist (mode '(emacs-lisp-mode lisp-interaction-mode))
+      (spacemacs/declare-prefix-for-mode mode "m=" "srefactor")
+      (evil-leader/set-key-for-mode mode
+        "m=b" 'srefactor-lisp-format-buffer
+        "m=d" 'srefactor-lisp-format-defun
+        "m=o" 'srefactor-lisp-one-line
+        "m=s" 'srefactor-lisp-format-sexp))))
+
+(defun emacs-lisp/post-init-smartparens ()
+  (if (version< emacs-version "25")
+      (ad-disable-advice 'preceding-sexp 'around 'evil)
+    (advice-remove 'elisp--preceding-sexp 'evil--preceding-sexp))
```

Comment:
I'm having trouble figuring out when the new functions came into effect. Is it 24.4?